### DISCUSSION TITLE
Allow changing default option for Issues display

### DISCUSSION THREAD

**Discussion by qlquanle** [community/community#4756] [Mobile] [answered]
Currently, Issues is opened to "Created" by default. Would it be possible to add an option to have it opened to "Assigned" by default? Thanks!

**Comment by brianlovin** [comment, marked as answer]
Thanks for the note! We're going to add the ability for filters to persist across views and sessions, so that if you prefer to have it set to `Assigned` by default, it will remember that choice. I can't give an exact estimate on when this will ship, but we're already beginning the work on it 😊

**Reply by qlquanle** [reply]
Brilliant, thank you!
